torch/library.py

```diff
@@ -257,14 +257,15 @@ def define(qualname, schema, *, lib=None, tags=()):
         >>> torch.library.define("mylib::sin", "(Tensor x) -> Tensor")
         >>>
         >>> # Add implementations for the operator
-        >>> @torch.library.impl("mylibrary::sin", "cpu")
+        >>> @torch.library.impl("mylib::sin", "cpu")
         >>> def f(x):
         >>>     return torch.from_numpy(np.sin(x.numpy()))
         >>>
         >>> # Call the new operator from torch.ops.
         >>> x = torch.randn(3)
-        >>> y = torch.ops.mylib.sin(x)
-        >>> assert torch.allclose(y, x)
+        >>> y1 = torch.ops.mylib.sin(x)
+        >>> y2 = np.sin(x)
```

Comment:
should be y2 = torch.sin(x) ?
I can believe this code (calling np.sin on a PyTorch tensor) works out of the box (I haven't tested it out), but to avoid confusion torch.sin is better.

Reply:
Good catch. Fixed this and other instances in latest commit. Thanks.